### PR TITLE
Travis files added

### PR DESCRIPTION
**Pull request by akorosmezey**

### CODE REVIEW

File: ci [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -ev # Ref https://docs.travis-ci.com/user/customizing-the-build/#Implementing-Complex-Build-Steps
+set -x
+
+case "${1:?}"-"${2:?}" in
+    before_install-tests)
```
Comment (lucafavatella) [inline ci]: Dead code.

File: ci [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        ;;
+    script-dialyzer)
+        BuildDir="${3:?}"
+        ( cd "${BuildDir:?}" && make dialyzer; )
+        ;;
+    script-xref)
+        BuildDir="${3:?}"
+        ( cd "${BuildDir:?}" && ./rebar3 xref; )
+        ;;
+    after_failure-tests)
```
Comment (lucafavatella) [inline ci]: Dead code.

File: ci [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        BuildDir="${3:?}"
+        ls -l "${BuildDir:?}"/_build/dev3/rel/ae_core/log/*
+        for F in "${BuildDir:?}"/_build/dev3/rel/ae_core/log/*; do echo "${F:?}"; cat "${F:?}"; echo; done
+        ls -l "${BuildDir:?}"/_build/dev1/rel/ae_core/log/*
+        for F in "${BuildDir:?}"/_build/dev1/rel/ae_core/log/*; do echo "${F:?}"; cat "${F:?}"; echo; done
+        ls -l "${BuildDir:?}"/_build/dev2/rel/ae_core/log/*
+        for F in "${BuildDir:?}"/_build/dev2/rel/ae_core/log/*; do echo "${F:?}"; cat "${F:?}"; echo; done
+        ;;
+    after_failure-tester)
```
Comment (lucafavatella) [inline ci]: Dead code.

Comment (akorosmezey) [reply ci]: fixed now

File: ci [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        BuildDir="${3:?}"
+        ( cd "${BuildDir:?}" && make dialyzer; )
+        ;;
+    script-xref)
+        BuildDir="${3:?}"
+        ( cd "${BuildDir:?}" && ./rebar3 xref; )
+        ;;
+    after_failure-test)
+        BuildDir="${3:?}"
+        ls -l "${BuildDir:?}"/_build/dev3/rel/ae_core/log/*
```
Comment (lucafavatella) [inline ci]: I understand the dev3/dev1/dev2 nodes are not generated so I understand this can be deleted.

**Comment by sennui** [comment]
please remove all references to old devN nodes and squash commits

**Comment by zp-sd** [comment]
Can we reference issue numbers in branch name + commit msgs?

**Pull request merged**